classes/class-pods-beaver-page-data.php

```diff
@@ -515,18 +537,28 @@ public static function pods_get_templates() {
 	 */
 	public static function pods_get_settings_fields( $field_options = array() ) {
 
-		$pod_names = (array) pods_api()->load_pods( array( 'type' => array( 'settings', 'user' ), 'names' => true ) );
+		$pod_names = (array) pods_api()->load_pods( array( 'type' => array( 'user', 'settings' ), 'names' => true ) );
 
 		$field_options['add_pod_name'] = 'true';
 
 		$fields = array(
 			'settings_field' => array(
 				'type'    => 'select',
-				'label'   => __( 'Settings or User Field', 'pods-beaver-builder-themer-add-on' ),
+				'label'   => __( 'Settings, Author, User...', 'pods-beaver-builder-themer-add-on' ),
```

Comment:
Use a proper ellipsis: …

Reply:
Nice one didn't hink about it … did replace all ... with … thanks for taking a look